tests/integration-service/integration.go

```diff
@@ -201,6 +217,18 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 			Expect(f.AsKubeDeveloper.IntegrationController.WaitForAllIntegrationPipelinesToBeFinished(testNamespace, applicationName, snapshot)).To(Succeed())
 		})
 
+		It("checks if the status of integration test si reported in the Snapshot", func() {
```

Comment:
tiny typo: `checks if the status of integration test si reported in the Snapshot` -> `checks if the status of integration test is reported in the Snapshot`